flink-runtime/src/main/java/org/apache/flink/runtime/blob/BlobServer.java

```diff
@@ -426,4 +440,11 @@ void unregisterConnection(BlobServerConnection conn) {
 		}
 	}
 
+	/* Secure cookie to authenticate */
+	@Override
+	public String getSecureCookie() { return secureCookie; }
+
+	/* Flag to indicate if service level authentication is enabled or not */
+	public boolean isSecurityEnabled() { return secureCookie != null; }
```

Comment:
We expect the secure cookie configuration to be available if security is enabled. A missing value will be reported as an error ahead of time. Are you expecting any other conditions to be met? Could you please elaborate?